<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Doctor CRUD</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/equipo-medico.png</normaloff>icons/equipo-medico.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>701</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Datos</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>311</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="fullnameLabel">
       <property name="text">
        <string>Nombre:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="emailLabel">
       <property name="text">
        <string>Correo:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="telephoneLabel">
       <property name="text">
        <string>Telefono:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="OfficeLabel">
       <property name="text">
        <string>Consultorio:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="fullnameInput"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="emailInput"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="telephoneInput"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="officeInput"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="specialtyLabel">
       <property name="text">
        <string>Especialidad:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="professionalLicenseLabel">
       <property name="text">
        <string>Cedula Profesional:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="specialtyInput"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="professionalLicenseInput"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="stateLabel">
       <property name="text">
        <string>Estado:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="stateInput"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>321</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="removeButton">
       <property name="text">
        <string>Borrar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="createButton">
       <property name="text">
        <string>Crear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="modifyButton">
       <property name="text">
        <string>Modificar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QTableView" name="doctorsTable">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>701</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
